### DISCUSSION TITLE
How do I change DaysOfWeek TextColor?

### DISCUSSION THREAD

**Discussion by diegomotti** [ME-MarvinE/XCalendar#188] [Q&A] [answered]
Can I set a style for DaysOfWeek labels (Mon,Tue,Wed...) in XCalendar?
Actually its always Black.

**Comment by ME-MarvinE** [comment, marked as answer]
Currently this is only supported through templating since the default template is just a label.

The CalendarView's `DayNameTemplate` property can be set to a `DataTemplate` with a Label inside it. Then you just change the Label colour.

Here's a code snippet of the `DataTemplate` the MAUI CalendarView uses by default:
```xaml
 <DataTemplate x:DataType="{x:Type System:DayOfWeek}">
     <Label
        FontSize="15"
        HorizontalTextAlignment="Center"
        Text="{Binding ., Converter={StaticResource.LocalizeDayOfWeekAndCharLimitConverter}, ConverterParameter=3}"
        TextColor="Black"
        VerticalTextAlignment="Center"/>
</DataTemplate>
```

`LocalizeDayOfWeekAndCharLimitConverter` is a `ValueConverter` residing in XCalendar.Maui.Converters that has been declared as a resource outside the snippet. You can also use the `StringCharLimitConverter` if you don't care about localisation.


Issue #29 has more information/examples, and issue #165 may also be useful to look at.

Let me know if you have any more questions :)

**Comment by diegomotti** [comment]
Thanks! It's worked.
